mmv1/third_party/terraform/services/compute/resource_compute_instance.go.tmpl

```diff
@@ -1091,19 +1098,26 @@ be from 0 to 999,999,999 inclusive.`,
 						"enable_nested_virtualization": {
 							Type:        schema.TypeBool,
 							Optional:    true,
-							AtLeastOneOf: []string{"advanced_machine_features.0.enable_nested_virtualization","advanced_machine_features.0.threads_per_core"},
+							AtLeastOneOf: advancedMachineFeaturesKeys,
```

Comment:
I'm assuming it's Ok to require just at least one of these (or even an empty block might be Ok?

I didn't see obvious evidence of one of them requiring another, but it's possible the rules could be a little more complex here?

Reply:
I would keep it to be safe. It looks like this was only added to prevent an empty block from being specified (see https://github.com/GoogleCloudPlatform/magic-modules/pull/4849#discussion_r659056546), and perhaps we don't technically need it, but if the behavior of `advanced_machine_features: {}` is no different than not specifying it, then it's better to enforce the more clear option (not specifying).